dataprofiler/profilers/utils.py

```diff
@@ -275,4 +275,63 @@ def add_nested_dictionaries(first_dict, second_dict):
         if item not in first_dict:
             merged_dict[item] = copy.deepcopy(second_dict[item])
 
-    return merged_dict
\ No newline at end of file
+    return merged_dict
+
+def biased_skew(df_series):
+    """
+    Calculates the biased estimator for skewness of the given data.
+    The definition is formalized as g_1 here:
+        https://en.wikipedia.org/wiki/Skewness#Sample_skewness
+    :param df_series: data to get skewness of, assuming floats
+    :type df_series: pandas Series
+    :return: biased skewness
+    :rtype: float
+    """
+    n = len(df_series)
+    if n < 1:
+        return np.nan
+
+    mean = sum(df_series) / n
+    diffs = df_series - mean
+    squared_diffs = diffs ** 2
+    cubed_diffs = squared_diffs * diffs
+    M2 = sum(squared_diffs)
+    M3 = sum(cubed_diffs)
+    # Zero out if possible to prevent floating point errors
+    M2 = 0 if np.abs(M2) < 1e-14 else M2
+    M3 = 0 if np.abs(M3) < 1e-14 else M3
+
+    if (M2 == 0):
+        return 0.0
```

Comment:
in this case, should we return np.nan as does scipy?

Reply:
or does scipy return 0 in this case?

Reply:
If M2 is zero, then the variance of the data must be zero, implying all elements must be the same value. This doesn't mean that the skewness/kurtosis is undefined, but rather zero as well. Scipy/pandas return zero as well.

Reply:
what about the float point issue for small numbers as pointed out by pandas in their calc?

Reply:
Pandas applies the function [_zero_out_fperr](https://github.com/pandas-dev/pandas/blob/b4dcf3b535d06dd3d597e37be46a4f178fe6b542/pandas/core/nanops.py#L1496) to M2/M3/M4 after computing them. The function essentially returns 0 if the given value is less than 1e-14, otherwise it returns the original value, which is what I also correct for.

Reply:
let's add a comment referencing where the idea comes form, it's good we commented it, but we should mention the idea is derived bc pandas experiences with issues